ethical_statement.md

```diff
@@ -0,0 +1,35 @@
+## Preamble
+
+OpenDataCam (ODC) was created as an open-source tool to quantify the world. It is designed to be an accessible, affordable and open-source solution to better understand interactions in urban environments.
```

Comment:
I think we should put this broader. ODC was also used to count fish (which I personally believe is a perfectly all right use case)

So maybe follow the statement of the website around "quantifying the world"?